src/users/schemas/user-settings.schema.ts

```diff
@@ -74,12 +71,12 @@ export class UserSettings {
 
   @ApiProperty({
     type: [Object],
-    default: FILTER_CONFIGS,
+    default: [],
     description: "Array of filters the user has set",
   })
   @Prop({
     type: [{ type: Object }],
-    default: FILTER_CONFIGS,
+    default: [],
   })
   filters: FilterConfig[];
```

Comment:
FilterConfigType needs to be changed.
It should be similar to:
```suggestion
export type FilterComponentType =
  | "LocationFilterComponent"
  | "PidFilterComponent"
  | "PidFilterContainsComponent"
  | "PidFilterStartsWithComponent"
  | "GroupFilterComponent"
  | "TypeFilterComponent"
  | "KeywordFilterComponent"
  | "DateRangeFilterComponent"
  | "TextFilterComponent";

// Define the Filter interface
export interface FilterConfig {
  FilterComponentType: string;
  visible: boolean;
}
```